src/thrift/Makefile

```diff
@@ -27,7 +27,11 @@ $(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
 	# (https://jira.apache.org/jira/browse/THRIFT-3650)
 	patch -p1 < ../patch/0001-Revert-THRIFT-3650-incorrect-union-serialization.patch
 	patch -p1 < ../patch/0002-cve-2017-1000487.patch
+ifeq ($(CROSS_BUILD_ENVIRON), y)
```

Comment:
libthrift deb in itself is building properly, but the *-install is failing. seems there is some dependency missing. 

```
[ FAIL LOG START ] [ target/debs/buster/libthrift-0.11.0_0.11.0-4_arm64.deb-install ]
Selecting previously unselected package libthrift-0.11.0:arm64.
(Reading database ... 183765 files and directories currently installed.)
Preparing to unpack .../libthrift-0.11.0_0.11.0-4_arm64.deb ...
Unpacking libthrift-0.11.0:arm64 (0.11.0-4) ...
dpkg: dependency problems prevent configuration of libthrift-0.11.0:arm64:
 libthrift-0.11.0:arm64 depends on libgcc1-arm64-cross (>= 1:3.0).

dpkg: error processing package libthrift-0.11.0:arm64 (--install):
 dependency problems - leaving unconfigured
Processing triggers for libc-bin (2.28-10) ...
Errors were encountered while processing:
 libthrift-0.11.0:arm64
[  FAIL LOG END  ] [ target/debs/buster/libthrift-0.11.0_0.11.0-4_arm64.deb-install ] 
```